build.gradle

```diff
@@ -260,7 +261,8 @@ task downloadDependencies(type: Exec) {
     text += " && mvn install:install-file -Dfile=spigot-1.12.jar -DgroupId=org.spigotmc -DartifactId=spigot112R1 -Dversion=1.12.0-R1.0 -Dpackaging=jar"
     text += " && mvn install:install-file -Dfile=spigot-1.13.jar -DgroupId=org.spigotmc -DartifactId=spigot113R1 -Dversion=1.13.0-R1.0 -Dpackaging=jar"
     text += " && mvn install:install-file -Dfile=spigot-1.13.2.jar -DgroupId=org.spigotmc -DartifactId=spigot113R2 -Dversion=1.13.2-R2.0 -Dpackaging=jar"
-    text += " && mvn install:install-file -Dfile=spigot-1.14.2.jar -DgroupId=org.spigotmc -DartifactId=spigot114R1 -Dversion=1.14.1-R1.0 -Dpackaging=jar"
+    text += " && mvn install:install-file -Dfile=spigot-1.14.2.jar -DgroupId=org.spigotmc -DartifactId=spigot114R1 -Dversion=1.14.2-R1.0 -Dpackaging=jar"
```

Comment:
Remove the the installing of 1.14.2. We do only need 1.14.4.

Reply:
Okay I'll fix